skimage/metrics/tests/test_structural_similarity.py

```diff
@@ -257,3 +261,28 @@ def test_invalid_input():
         structural_similarity(X, X, K2=-0.1)
     with pytest.raises(ValueError):
         structural_similarity(X, X, sigma=-1.0)
+
+
+def test_ms_structural_similarity():
+    # color image example
+    Xc = data.chelsea()
+    sigma = 15.0
+    Yc = np.clip(Xc + sigma * np.random.randn(*Xc.shape), 0, 255)
+    Yc = Yc.astype(Xc.dtype)
+
+    # multichannel result should be mean of the individual channel results
+    mssim = muliscale_structural_similarity(Xc, Yc, channel_axis=-1)
+    mssim_sep = [
+        muliscale_structural_similarity(Yc[..., c], Xc[..., c])
+        for c in range(Xc.shape[-1])
+    ]
+    assert_almost_equal(mssim, np.mean(mssim_sep))
+
+    # structural_similarity of image with itself should be 1.0
+    assert_equal(muliscale_structural_similarity(Xc, Xc, channel_axis=-1), 1.0)
+
+    # test small image
+
+    X = np.zeros((5, 5))
```

Comment:
Could this be a less-trivial test? Let's say, an identity matrix?

Reply:
Thanks for the suggestion! I will do it soon.

Reply:
oh I realized this test was meant to be trivial. It should raise an `AssertionError` that the images are too small. So I think as no calc is done it doesn't matter.